packages/contracts-bedrock/src/L1/opcm/OPContractsManagerV2.sol

```diff
@@ -803,9 +839,46 @@ contract OPContractsManagerV2 is ISemver, OPContractsManagerUtilsCaller {
             abi.encodeCall(IDelayedWETH.initialize, (_cts.systemConfig))
         );
 
+        // Update the ETHLockbox, DisputeGameFactory, and AnchorStateRegistry.
+        // If this is an interop migration upgrade, these contracts may have a different
+        // ProxyAdmin, but regardless all chains migrating to an interop set MUST have the
+        // same ProxyAdmin owner (which is delegate calling this contract).
+        // Therefore for the next set of upgrades, we use either:
+        //   1. The current chain's proxy admin for initial deployments
+        //   2. The ETHLockbox's proxyAdmin() getter for upgrades and interop migrations (if lockbox exists)
+        //   3. The current chain's proxy admin for upgrades when lockbox doesn't exist
+        // It would be ideal to simply use the proxyAdmin() getter in all cases, however for initial deployments
+        // we are unable to do so, because the implementation address is not yet set.
+        IProxyAdmin interopContractsProxyAdmin = _isInitialDeployment || address(_cts.ethLockbox) == address(0)
```

Comment:
Nit: worth updating this comment to reflect the fact that if the chain is not using interop, this logic doesn't do anything special and has no effect (because the ethLockbox proxy admin is equivalent to `_cts.proxyAdmin`.